presto-docs/src/main/sphinx/connector/snowflake.rst

```diff
@@ -0,0 +1,111 @@
+===============
+Snowflake Connector
+===============
+
+The Snowflake connector allows querying and creating tables in an external
+Snowflake database. This can be used to join data between different
+systems like Snowflake and Hive, or between two different Snowflake accounts.
+
+Configuration
+-------------
+
+To configure the Snowflake connector, create a catalog properties file
+in ``etc/catalog`` named, for example, ``snowflake.properties``, to
+mount the Snowflake connector as the ``snowflake`` catalog.
+Create the file with the following contents, replacing the
+connection properties as appropriate for your setup:
+
+.. code-block:: none
+
+    connector.name=snowflake
+    connection-url=jdbc:snowflake://<account_identifier>.snowflakecomputing.com/?db={dbname}
+    connection-user=root
+    connection-password=secret
+
+The ``connection-url`` defines the connection information and parameters to pass
+to the Snowflake JDBC driver.
+
+.. code-block:: text
+
+    connection-url=jdbc:snowflake://<account_identifier>.snowflakecomputing.com/?db={dbname}
+
+The ``connection-user`` and ``connection-password`` are typically required and
+determine the user credentials for the connection, often a service user.
+
+Multiple Snowflake Databases or Accounts
+^^^^^^^^^^^^^^^^^^^^^^
+
+The Snowflake connector can only access a single database within a Snowflake account.
+Thus, if you have multiple Snowflake databases, or want to connect to multiple
+Snowflake accounts, you must configure multiple instances of the Snowflake connector.
+To add another catalog, simply add another properties file to ``etc/catalog``
```

Comment:
```suggestion
To add another catalog, add another properties file to ``etc/catalog``
```
If documentation describes something as simple or easy, and the user doesn't feel the same way, they can feel excluded. Avoid simple or easy. See the [GitLab Documentation Style Guide entry for simple](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html#simply-simple) and the [Vale linter simplicity.yml](https://gitlab.com/gitlab-org/gitlab/-/blob/master/doc/.vale/gitlab/Simplicity.yml) for related words.